pom.xml

```diff
@@ -22,27 +22,39 @@
   <parent>
     <groupId>io.smallrye</groupId>
     <artifactId>smallrye-parent</artifactId>
-    <version>8</version>
+    <version>10</version>
   </parent>
 
-  <artifactId>smallrye-metrics-parent-2.0</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <artifactId>smallrye-metrics-parent</artifactId>
+  <version>2.1.1-SNAPSHOT</version>
 
   <packaging>pom</packaging>
   <name>SmallRye: MicroProfile Metrics Parent</name>
+  <url>http://smallrye.io</url>
 
   <properties>
     <version.microprofile-config>1.3</version.microprofile-config>
     <version.microprofile-metrics>2.0.1</version.microprofile-metrics>
 
-    <version.smallrye.config>1.0.0</version.smallrye.config>
+    <version.smallrye.config>1.3.9</version.smallrye.config>
 
     <version.jboss.servlet>1.0.2.Final</version.jboss.servlet>
 
-    <!-- Used by release plugin to define git tag -->
-    <tagNameFormat>smallrye-metrics-2.0-@{project.version}</tagNameFormat>
+    <sonar.coverage.jacoco.xmlReportPaths>${project.basedir}/testsuite/rest-tck/target/site/jacoco-aggregate/jacoco.xml,${project.basedir}/../../testsuite/rest-tck/target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
This should include the `implementation` module which is the only module which can measure code coverage at the moment (the others are Arquillian tests running in different JVMs so until we figure out a way to properly configure the JaCoCo agent there we will get just zeros)

Reply:
By the way I've spent like two whole days already fighting with JaCoCo and Sonar to get coverage from the TCKs, right now I'm finally able to measure the coverage, but when it uploading to Sonar, coverage from these tests is skipped probably because the sources of the TCK tests are not present in the repository (they are a dependency, which apparently Sonar doesn't expect).

Reply:
It doesn't need to include the `implementation` module as the testsuite aggregates the jacoco results

Reply:
Oh I see now, if I run `mvn verify -Pcoverage` then I look at `testsuite/rest-tck/target/site/jacoco-aggregate/smallrye-metrics/index.html` I see about 49 % coverage (of instructions, and I think this coverage is only from the `integration` module) but when I later do `mvn sonar:sonar` with my credentials, then in Sonarcloud I see 0 % (and the same 0 % from the upload created from this PR[1]), not sure why exactly...

Also could we get the aggregation done rather into the root module rather than one particular TCK module?

[1] https://sonarcloud.io/project/activity?graph=coverage&id=smallrye_smallrye-metrics&selected_date=2019-08-09T19%3A22%3A44%2B0000

Reply:
Weird, the same settings have worked in other projects.

I've just pushed a change to use a different path for finding the sonar output, so will see what happens.

I haven't had any luck in doing the aggregation in the root, but you're more than welcome to give it a go once this is in. Other projects are also doing it within the testuite